scico/test/linop/test_linop.py

```diff
@@ -519,49 +519,6 @@ def sumtestobj(request):
     yield SumTestObj(request.param)
```

Comment:
Is this fixture still used by something?

Reply:
Good catch, removed.